atomic_reactor/plugins/pre_pin_operator_digest.py

```diff
@@ -318,6 +260,23 @@ def run_in_worker(self):
         else:
             self.log.warning("%s has a relatedImages section, skipping", operator_csv.path)
 
+        return operator_manifests_metadata
+
+    def should_run(self):
+        """
+        Determine if this is an operator manifest bundle build
+
+        :return: bool, should plugin run?
+        """
+        if not has_operator_bundle_manifest(self.workflow):
+            self.log.info("Not an operator manifest bundle build, skipping plugin")
+            return False
+        if not self.workflow.conf.operator_manifests:
+            msg = "operator_manifests configuration missing in reactor config map, aborting"
+            self.log.warning(msg)
+            return False
```

Comment:
I wonder wonder shouldn't this actually fail? I mean you want to use feature so you expect it to work, but this way we are just silently doing nothing if it isn't configured

@chmeliik @MartinBasti

Reply:
Hmm, hard to say. The current behavior is not ideal, but immediately failing if the configuration is missing may not be ideal either. Users have ways to do the pinning themselves and tell OSBS to skip processing, so maybe that should be evaluated before checking for server config and raising errors.

TL;DR, I don't think it should be fixed in this PR